### PR TITLE
Fix failing cypress tests

### PR DESCRIPTION
**Pull request by Larisa-Staroverova**

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts [hunk 1]
```diff
@@ -231,6 +231,14 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
       value: 'Resolve Issues',
       type: TabbingOrderTypes.LINK,
     },
+    {
```
Comment (ChristophHi) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts]: [product-configurator-tabbing.core-e2e-spec-flaky.ts] fails with standard backend https://40.76.109.9:9002". Our tests should be able to run with different backend versions, correct? TBD how we achieve that, in other suites we introduced a backend release setting

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc.e2e-2205-spec-flaky.ts [hunk 2]
```diff
@@ -91,13 +91,15 @@ context('Product Configuration - 2205', () => {
       cy.wait('@updateConfig');
```
Comment (ChristophHi) [inline projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc.e2e-2205-spec-flaky.ts:91]: [product-configurator-vc.e2e-2205-spec-flaky.ts] is failing for me with a current mock backend
Screenshot 2023-01-04 at 15 54 01

**Review by ChristophHi — CHANGES_REQUESTED** [review]

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts (hunk 1 above)
Comment (Larisa-Staroverova) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts]: Done!

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc.e2e-2205-spec-flaky.ts (hunk 2 above)
Comment (Larisa-Staroverova) [reply projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc.e2e-2205-spec-flaky.ts:91]: Tried to stabilize this test. Done!

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc.e2e-2205-spec-flaky.ts [hunk 3]
```diff
@@ -52,7 +52,7 @@ context('Product Configuration - 2205', () => {
     configUISettings.productConfigurator.enableNavigationToConflict = false;
   });
 
-  describe('Conflict Solver', () => {
+  describe.only('Conflict Solver', () => {
```
Comment (ChristophHi) [inline projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc.e2e-2205-spec-flaky.ts]:
```suggestion
  describe('Conflict Solver', () => {
```

**Review by ChristophHi — CHANGES_REQUESTED** [review]

**Review by ChristophHi — APPROVED** [review]

**Pull request merged**